### PR TITLE
Update cl_spectator_deathmatch.lua

### PR DESCRIPTION
**Pull request by markusmarkusz**
Say no to SetNoDraw!
Please test it. But it should work.

### CODE REVIEW

**Comment by MinIsMin** [comment]
Why using ipairs?

**Comment by markusmarkusz** [comment]
ipairs is for sequential tables faster. afaik it is because ipairs is jit-compiled.

File: lua/cl_spectator_deathmatch.lua [hunk 1]
```diff
@@ -62,48 +62,43 @@ hook.Add("RenderScreenspaceEffects", "RenderScreenspaceEffects_Ghost", function(
 	end
 end)
 
+local COLOR_WHITE = COLOR_WHITE
```
Comment (markusmarkusz) [inline lua/cl_spectator_deathmatch.lua]: Micro optimization. This probably brings very little, but often used things should be precached. You could also just precache the color. But calling Color() the whole time is terrible.

**Comment by MinIsMin** [comment]
You may want to fix the lila bandanas... xD

**Comment by markusmarkusz** [comment]
Change `local COLOR_WHITE = COLOR_WHITE` to `local COLOR_WHITE = Color(255, 255, 255, 255)`.
@GamefreakDE said that this fixes it. (I've never had a problem like this... Everything works fine for me.)

**Pull request merged**